argocd/metadata.csv

```diff
@@ -0,0 +1,132 @@
+argocd.api_server.process.cpu.seconds,count,,second,,The total user and system CPU time spent in seconds in the API Server,0,argocd,api_server process cpu seconds,
+argocd.api_server.process.max_fds,gauge,,,,The maximum number of open file descriptors in the API Server,0,argocd,api_server process max_fds,
+argocd.api_server.process.open_fds,gauge,,,,The number of open file descriptors in the API Server,0,argocd,api_server process open_fds,
+argocd.api_server.process.resident_memory.bytes,gauge,,byte,,The nesident memory size in bytes in the API Server,0,argocd,api_server process resident_memory bytes,
+argocd.api_server.process.start_time.seconds,gauge,,second,,The start time of the process since unix epoch in seconds in the API Server,0,argocd,api_server process start_time seconds,
+argocd.api_server.process.virtual_memory.bytes,gauge,,byte,,The virtual memory size in bytes in the API Server,0,argocd,api_server process virtual_memory bytes,
+argocd.api_server.process.virtual_memory.max_bytes,gauge,,byte,,The maximum amount of virtual memory available in bytes in the API Server,0,argocd,api_server process virtual_memory max_byte,
+argocd.api_server.redis.request,count,,request,,The number of kubernetes requests executed during application reconciliation in the API Server,0,argocd,api_server redis request,
+argocd.api_server.redis.request.duration,histogram,,request,,The Redis requests duration,0,argocd,api_server redis request duration,
+argocd.app_controller.app.info,gauge,,,,Information about Applications. It contains tags such as sync_status and health_status that reflect the application state in ArgoCD. The metric value is constant,0,argocd,app_controller app info,
+argocd.app_controller.app.k8s.request,count,,request,,The number of kubernetes requests executed during application reconciliation,0,argocd,app_controller app k8s request,
+argocd.app_controller.app.reconcile,histogram,,,,Application reconciliation performance,0,argocd,app_controller app reconcile,
+argocd.app_controller.app.sync,count,,,,The total number of application syncs,0,argocd,app_controller app sync,
+argocd.app_controller.cluster.api.resource_objects,gauge,,object,,The number of k8s resource objects in the cache,0,argocd,app_controller cluster api resource_objects,
+argocd.app_controller.cluster.api.resources,gauge,,resource,,The number of monitored kubernetes API resources.,0,argocd,app_controller cluster api resources,
+argocd.app_controller.cluster.cache.age.seconds,gauge,,second,,The age of the cluster cache in seconds,0,argocd,app_controller cluster cache age seconds,
+argocd.app_controller.cluster.connection.status,gauge,,,,The current k8s cluster connection status,0,argocd,app_controller cluster connection status,
+argocd.app_controller.cluster.events,count,,event,,The number of processed k8s resource events,0,argocd,app_controller cluster events,
+argocd.app_controller.cluster.info,,,,,      ,0,argocd,app_controller cluster info,
```

Comment:
This line seems to have a ton of extra space, is there something that should be here?

Reply:
Fixed!